packages/editor-ui/src/stores/workflows.ts

```diff
@@ -818,9 +827,6 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 
 			Vue.set(node, 'parameters', newParameters);
 
-			if (!this.nodeMetadata[node.name]) {
-				Vue.set(this.nodeMetadata, node.name, {});
-			}
 			Vue.set(this.nodeMetadata[node.name], 'parametersLastUpdatedAt', Date.now());
```

Comment:
nitpick - might be worth keeping the check before this..

Reply:
I've moved this to `addNode` action to make sure it's always there

Reply:
ya just out of redundancy..

Reply:
Alright, I've reverted it. Can you pls check&approve again?